odoo/addons/base/views/onboarding_views.xml

```diff
@@ -24,110 +24,68 @@
                 </div>
             </div>
         </div>
+
         <div class="o_onboarding_container collapse show">
-            <div t-att-class="'o_onboarding ' + ('o_onboarding_completed ' if steps_completed else ' ') + classes"
+            <div t-att-class="'o_onboarding_main position-relative overflow-hidden ' + classes"
                  t-att-style="'background-image:url(' + bg_image + ')'">
-                <div class="o_onboarding_wrap">
-                    <a href="#" data-toggle="modal" data-target=".o_onboarding_modal"
-                       class="float-right o_onboarding_btn_close">
-                        <i class="fa fa-times" title="Close the onboarding panel" />
-                    </a>
-                    <div class="o_onboarding_steps">
-                        <t t-out="0" />
+
+                <div class="o_onboarding_wrap py-5">
+                    <a href="#" data-toggle="modal" data-target=".o_onboarding_modal" class="o_onboarding_btn_close position-absolute top-0 end-0 py-2 px-3 h2 text-white" title="Close the onboarding panel"><i class="oi oi-close"/></a>
+
+                    <div class="o_onboarding_steps d-flex">
+                        <t t-out="0"/>
                     </div>
-                    <div t-att-class="'o_onboarding_completed_message text-center' + (' o_onboarding_steps_done' if state.get('onboarding_state') in ('done', 'just_done') else '') + (' o_onboarding_steps_just_done' if state.get('onboarding_state') == 'just_done' else '')">
-                        <i class="fa fa-check text-success mr8" />
-                        <span class="h3 mr16">
+
+                    <div t-if="state.get('onboarding_state') in ('done', 'just_done')"
+                         t-att-state="state.get('onboarding_state')"
+                         t-attf-class="o_onboarding_completed_message position-absolute end-0 bottom-0 left-0 border-bottom py-4 bg-white d-flex align-items-center justify-content-center">
```

Comment:
all those classes are static I think, so maybe just `class=` instead of `t-attf-class=` ?